hardware/DomoticzHardware.h

```diff
@@ -70,7 +70,7 @@ class CDomoticzHardwareBase : public StoppableTask
 		__attribute__((format(printf, 3, 4)))
 #endif
 		;
-
+	uint32_t m_LogLevelEnabled; //bitwise _eLogLevel 7 = LOG_NORM | LOG_STATUS | LOG_ERROR
```

Comment:
As this is a unsigned integer used as a bit-wise logical structure, shouldn't we make it a little smaller? We only use 3 bits now and reserving 32 is quite some room to grow :) We are not using these bits for hardware- or devicetypes (of which there are many). Maybe uint16_t is more than enough?

Reply:
Internally we already used a uint32 so I continued with it.. 😁
But there a possible issue with this PR that I need to check..  some error in hardware.js